exporter/prometheus/prometheus_test.go

```diff
@@ -15,12 +15,17 @@
 package prometheus
```

Comment:
Missing license header.

Reply:
Not sure what you mean here. Is it just a matter of updating the year from 2017 to 2018?

Reply:
@rakyll it exists already, this isn't a new file :)
@freddygv it looked like a new file, and Github's code review doesn't let us expand
files easily.